sensor/common/store/nodes.go

```diff
@@ -0,0 +1,12 @@
+package store
+
+import (
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/net"
+)
+
+// NodeWrap adds address information to nodes to forward to central
+type NodeWrap struct {
```

Comment:
I don't understand the design decision of exposing this _internal_ struct of the listeners rather than using a separate interface returning only `*storage.Node`, which is what is really used outside the listener. 

I.e.: I don't see `Addresses` being used anywhere other than in the listener. So why do they need to be exposed in `sensor/common`?

Most stores have the similar requirements. The way this is handled for other stores is to have a sensor-wide API for the store (defined in `sensor/common`), which exposes `storage.*` type structs. And have its implementation be internal to `sensor/kubernetes`, where you can use any additional information needed.

Take deployments, for example:

```go
// In sensor/common/store:
type DeploymentStore interface {
	Get(id string) *storage.Deployment
}

// In sensor/kubernetes/listener/resources
type DeploymentStore struct { ... }

func (ds *DeploymentStore) addOrUpdateDeployment(wrap *deploymentWrap) { ... }

// Get returns deployment for supplied id.
func (ds *DeploymentStore) Get(id string) *storage.Deployment {
	wrap := ds.getWrap(id)
        // parse to a storage.Deployment type to comply with public common interface
	return wrap.GetDeployment()
}
```